manifests/client.pp

```diff
@@ -57,6 +58,7 @@
   Optional[String]        $default_pool_full,
   Optional[String]        $default_pool_inc,
   Optional[String]        $default_pool_diff,
+  String                  $ensure              = 'present',
```

Comment:
An empty string is for sure not right, we can enforce it to be non-empty with:

```suggestion
  String[1]               $ensure              = 'present',
```

Same for other classes.

Reply:
Fixed. :)
(Not rebased on the other branch to keep things simple.)

Reply:
> (Not rebased on the other branch to keep things simple.)

Wanted to do this but I am not allowed to push in your branch:

```
ERROR: Permission to markt-de/puppet-bacula.git denied to smortex.
```

Reply:
rebased :)